cairo-contracts/packages/apps/src/transfer/components/transfer.cairo

```diff
@@ -781,12 +793,12 @@ pub mod TokenTransferComponent {
             self.write_ibc_token_address_to_key(token_address, denom_key);
         }
 
-        fn assert_non_ibc_token(
+        fn ibc_token_key(
```

Comment:
any reason why this returns the key now?

Reply:
Good catch! 
This is touched as I was trying to refactor the `denom` field too, but that didn't work and reverted. 
This ended up staying. Fixed: [08214b6](https://github.com/informalsystems/ibc-starknet/pull/235/commits/08214b652df8df8fe33ee560a8eda1220f7e8e91)